Add more technical and economical info to JLP and perpetuals guides (#479)
jup-ag/space-station: guides/7-jlp/2-How-JLP-Works.md

Traders pay these fees to the pool based on the amount of tokens they've borrowed. This mechanism helps secure the balance of the pool's assets and compensates liquidity providers for the use of their tokens.

### Add/Remove Liquidity into JLP

When a user adds liquidity to the JLP pool, they increase the amount of TVL (Total Value Locked) which increases the liquidity available for perps trading. The pool is currently a collection of the following tokens: **USDC, USDT, SOL, wBTC and ETH**.

**Adding Liquidity** 

Any Liquidity providers (LPs) can acquire JLP by swapping for it on Jupiter Swap. Jupiter Swap will find the best price to acquire JLP for you, automatically. This can either be purchasing it off the open market, or swapping it for a desired asset and depositing that into JLP directly. Use Jupiter Swap for a frictionless experience of getting JLP. While JLP is still being minted, your assets may be deposited into the relevant token pool increasing the current weightage. At the point of depositing assets into the JLP pool, the protocol will re-price the TVL in the USD value.

**Removing Liquidity** 

> JLP can also be sold via Jupiter Swaps. Simply swap JLP for your desired asset. The JLP token could either be transferred to another trader or may be redeemed by the JLP pool, burning the JLP token, and releasing some of the currency contained in the pool.

Author: ```suggestion
JLP can also be sold via Jupiter Swap. Simply swap JLP for your desired asset. The JLP token could either be transferred to another trader or may be redeemed by the JLP pool, burning the JLP token, and releasing some of the currency contained in the pool.
```

- Trading Fees of the Pool, for spot assets
- Minting and burning of JLP

> **75% of the fees generated by JLP go into the pool.** The fees are compounded into the pool weekly. 

Reviewer: hourly

| ------------------ | -------------------------------------------------------------------------- |
| Opening a Position | 6 BPS                                                                     |
| Closing a Position | 6 BPS                                                                     |
| Price Impact Fee   | Variable (see [reference](../8-perpetual-exchange/2-how-it-works.md#price-impact-fee)) |
| Swap Fee           | Between 0 BPS to 150 BPS depending on pool weightage                       |

> | Borrow Rate        | 0.8 BPS/hour x token utilization percentage                                  |

Author: Sorry I should have mentioned this earlier; can we update this to add 1 BPS / hour for stablecoins, in addition to the `0.8 BPS/hour` we have now?

Reviewer 2: I previously added this as an info portion below the table so the table wouldn't have too much words on it - is it ok?

> | Borrow Rate        | 0.8 BPS/hour x token utilization percentage                                  |

Reviewer: SOL, BTC and ETH are 0.8bps, USDC and USDT are 1bps

:::info

> **Hourly Borrow Rate** is at 0.008% for SOL, ETH and BTC while 0.01% for USDC and USDT.

Author: ```suggestion
**Hourly Borrow Rate** is at 0.008% for SOL, ETH and BTC and 0.01% for USDC and USDT.
```

estimated revenue share you generate = $22,500 x 0.025% = $5.625
```

### Risks Associated with Holding JLP

1. **Bull Markets**

During a bull market, JLP may not outperform SOL, ETH, or BTC. This is because the pool is made of a mix of stable and non-stable tokens. While this factor is less of a risk, it is worth noting that during a bull market, the JLP token will move slower than some of the coins it contains. 

> It is suggested to consider swapping **stable tokens** (USDC/USDT) into JLP to potentially benefit from yield generated by perpetual trading activities. Please note that this is **not financial advice**.

Reviewer 2: good point, will remove entirely - best not to influence user's decisions

estimated revenue share you generate = $22,500 x 0.025% = $5.625
```

### Risks Associated with Holding JLP

1. **Bull Markets**

> During a bull market, JLP may not outperform SOL, ETH, or BTC. This is because the pool is made of a mix of stable and non-stable tokens. While this factor is less of a risk, it is worth noting that during a bull market, the JLP token will move slower than some of the coins it contains. 

Author: ```suggestion
During a bull market, JLP may not outperform SOL, ETH, or BTC. This is because the pool is made of a mix of stable and non-stable tokens.
```

We can probably remove this since it's roughly the same as first two lines, WDYT?

**Short Trade Scenario:** If the trader profits on the short, the JLP pool will lose some of the stablecoins but the shorted token will remain the same. This causes a net USD loss on the Pool.

![image](jlp-short-scenarios.png)

*This [research article](https://skribr.io/app/article/exploring-jupiters-perpetual-futures-a-comprehensive-research-analysis/) from LeeWay provides more detail on how this works.*

3. **Token Price Fluctuations:** The JLP pool consists of both stable and non-stable tokens. Fluctuations in token prices can affect the value of JLP. As a result, users may find that their withdrawn tokens can be worth less compared to their initial deposit. Additionally, deposit and withdrawal fees for the JLP Pool may further reduce the number of tokens withdrawn, particularly for shorter holding periods.

**Can JLP go down?** Yes. As shown in the trader profiting on a short trade, JLP in USD value will go down when the fees generated are lower than depreciation of assets and payout from traders' profit.

### Pool Weightage

In the JLP pool, each asset is assigned a **target weight**, which is set by the team. 

![jlp-weightage](jlp-weightage.png)

> Experts like [Gauntlet](https://www.gauntlet.xyz/) and [Chaos Labs](https://chaoslabs.xyz/) conduct regular risk assessments and market analyses in response to the evolving market conditions, and discuss with the community before presenting to the team for rollout. 

Author: ```suggestion
Jupiter works with [Gauntlet](https://www.gauntlet.xyz/) and [Chaos Labs](https://chaoslabs.xyz/) to conduct regular risk assessments and market analyses in response to evolving market conditions, and discuss with the community before presenting to the team for rollout. 
```

I think we can change the first part to say `Jupiter works with...`; do you mind helping me change the last part of the paragraph as well? Since it reads kinda weird now that I've changed the first line, and IDK how to make it sound better 😅
